use-timescale/tigerlake.md

```diff
@@ -0,0 +1,251 @@
+   1. Click `Next`, then give the inline policy a name and click `Create policy`.
+
+   Provide $COMPANY with the ARN of this role, the ARN of the S3 table bucket.
+  $COMPANY uses the outputs to spin up your $LAKE_LONG services, then let you know when the $SERVICE_SHORT is 
+  ready to use. Provisioning takes a couple of minutes, during this time the $SERVICE_SHORT is restarted.
+
+</Procedure>
+
+</Tab>
+
+</Tabs>
+
+## Stream data from your $SERVICE_LONG to your data lake
+
+To stream data from a $PG relational table, or a $HYPERTABLE in your $SERVICE_LONG to your data lake, run the following 
+statement:
+
+```sql
+ALTER TABLE <table_name> SET (
+  tigerlake.iceberg_sync = true | false,
+  tigerlake.iceberg_partitionby = '<partition_specification>'
+)
+```
+
+* `tigerlake.iceberg_sync`: `boolean`, set to `true` to start streaming, or `false` to stop the stream. A stream 
+  **cannot** resume after being stopped. 
+* `tigerlake.iceberg_partitionby`: optional property to define a partition specification in Iceberg. By default the 
+  partitioning specification of the $HYPERTABLE is used. Streamed $PG tables can have a partition specification 
+  for the Iceberg table, if intentially defined. Please refer to [partitioning](#partitioning) for more details.
+
+Only tables or $HYPERTABLEs with primary keys are supported, this includes composite primary keys as well. 
+A primary key is necessary for Iceberg to perform update or delete statements.
+
+When a stream is started, the full table is synchronized to Iceberg, this means that all prior records are imported first.
+The write throughput is ranging at approximately 40.000 records / second, for larger tables a full import can take some time.
+
+### Partitioning
+
+By default, the partition interval for an Iceberg table is the same as the one from a $HYPERTABLE.
+The sync of a Postgres table does not enable any partitioning in Iceberg, but can be set through the [API](#api) with `tigerlake.iceberg_partitionby`.
+
+The following partition intervals and specifications are supported, and the define behavior of [Iceberg partition specification][iceberg-partition-spec].
+
+| Interval      | Description | Source types | Result type |
+| ------------- | --- | --- | --- |
+| `hour`        | Extract a date or timestamp day, as days from 1970-01-01 | `date`, `timestamp`, `timestamptz` | `int` |
+| `day`         | Extract a date or timestamp day, as days from 1970-01-01 | `date`, `timestamp`, `timestamptz` | `int` |
+| `month`       | Extract a date or timestamp day, as days from 1970-01-01 | `date`, `timestamp`, `timestamptz` | `int` |
+| `year`        | Extract a date or timestamp day, as days from 1970-01-01 | `date`, `timestamp`, `timestamptz` | `int` |
+| `truncate[W]` | Value truncated to width W, see [options][iceberg-truncate-options] | `int`, `long`, `decimal`, `string`, `binary` | `int` |
+
```

Comment:
I think it is good to add a section with examples here:

```suggestion
##Examples
This SQL enables iceberg sync on my_hypertable and sets up a daily partitioning scheme on the hypertable’s time column.  (i.e. equivalent to “day(ts_column)” ). 


ALTER TABLE my_hypertable SET (tigerlake.iceberg_sync = true)



You can explicitly specify the partitioning scheme using:


ALTER TABLE my_hypertable SET (tigerlake.iceberg_sync = true,
                                                          tigerlake.iceberg_partitionby =  ‘hour(ts_column)’

                                                          );


You can also setup Iceberg sync for any Postgres table.

ALTER TABLE my_postgres_table SET ( tigerlake.iceberg_sync = true);



```

Reply:
I had a look, and all of these examples are already in the page.

Reply:
I don't see these examples on the page. I only see the bare syntax.

Reply:
Let's add some examples, especially with regards to PG tables and partitioning. I'll accept the commit and polish these